CHANGES.md

```diff
@@ -29,6 +29,11 @@ OpenSSL 3.4
 
 ### Changes between 3.3 and 3.4 [xx XXX xxxx]
 
+ * Add debuginfo makefile target for unix platofms to produce
+   a separated DWARF info file from the corresponding shared libs
```

Comment:
s/separated/separate/ sounds better to me, also please add a dot at the end of the sentence.